frontend/src/column.tsx

```diff
@@ -22,7 +22,8 @@ export function Column(props: ColumnProps) {
          </Flex>
          <Box display={open ? 'block' : 'none'}>
             {props.pulls.map((pull) =>
-               <PullCard key={pull.getKey()} pull={pull}/>
+               // show is passed at top-level so PullCard is still memoizable
+               <PullCard key={pull.getKey()} pull={pull} show={pull.show}/>
```

Comment:
This is a little annoying- I appreciate the comment!

May be able to get around passing show as a separate prop by avoiding mutable state, but it might not be worth the performance hit: 
https://github.com/iFixit/pulldasher/pull/262#discussion_r834448510

Reply:
I decided on a different strategy; passing around a Set instead of mutating state.